src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/RenameDriver.cs

```diff
@@ -19,13 +18,15 @@ public sealed class RenameDriver
         private readonly RecordType _baseParameters;
         private readonly RecordType _renameParameters;
         private readonly INameResolver _resolver;
+        private readonly INameResolver _renameResolver;
```

Comment:
What the difference between these 2 INameResolvers?

Reply:
the second one is used for renaming option sets